pkg/kernel/kernel.go

```diff
@@ -284,7 +287,7 @@ func (k Kernel) doEval(code string) (val []interface{}, err error) {
 	job, err := k.Manager.Execute(code)
 	if err != nil {
 		fmt.Printf("Failed to create job: %v", err)
-		return nil, err
+		return nil, nil
```

Comment:
why returning nil here?

Reply:
To keep consistency with golang kernel, and for the execcounter. We have already logged the error.